src/fcgen/fastcat_types.yaml

```diff
@@ -106,6 +106,8 @@ states:
       type: uint8_t
     - name: target_reached
       type: uint8_t
+    - name: setpoint_ack_rise
+      type: bool
```

Comment:
I checked the needed modifications in Fastcat to accommodate the two different states, and they seem acceptable.

However, I also checked `ros2/fcat`, `ros1/fcat`, and EELS' `c0_mech`, and the impact of these changes there **and the applications that interact with them** will be significant. First, there will be significant code duplication when reading in the actuator state (e.g. ros1/fcat's [PublishJointStates](https://fornat1.jpl.nasa.gov/casah/ros1/fcat/-/blob/master/src/fcat.cpp#L335-359)). Second and most importantly, since this change implies that now two different fcat_msgs messages will be needed (`fcat_msgs::ActuatorState` -> `fcat_msgs::EgdActuatorState` and `fcat_msgs::EpdActuatorState`), virtually all clients of CASAH will be broken after the change and will require updating topic channel names, Fastcat state types, message types, included message header files, and their build configurations.

I personally think it is more preferable to have a single actuator state with potentially some extra fields that only the Platinum would populate (currently 1 and maybe 2-3 in the future). This would follow the spirit of an actuator state that belongs to a DS-402 drive, as opposed to a particular Elmo line, and would not break the existing code-base around CASAH.

@alex-brinkman, if you still want to go ahead with splitting actuator state, I can do it, but just be aware of the implications mentioned above.